test/unittests/execute_floating_point_test.cpp

```diff
@@ -92,7 +92,7 @@ struct WasmTypeName
 template <typename T>
 class execute_floating_point_types : public testing::Test
 {
-protected:
+public:
```

Comment:
why?

Reply:
I want to access the list of NaNs from other test suite.